### PR TITLE
cleanup header

### PR DESCRIPTION
**Pull request by crptm**

### CODE REVIEW

File: common/actions/config.js [hunk 1]
```diff
@@ -1,22 +1,12 @@
 // @flow
 export const CONFIG_LANGUAGE_CHANGE = 'CONFIG_LANGUAGE_CHANGE';
-export const CONFIG_LANGUAGE_DROPDOWN_TOGGLE = 'CONFIG_LANGUAGE_DROPDOWN_TOGGLE';
 export const CONFIG_NODE_CHANGE = 'CONFIG_NODE_CHANGE';
-export const CONFIG_NODE_DROPDOWN_TOGGLE = 'CONFIG_NODE_DROPDOWN_TOGGLE';
 
 
-export const CHANGE_LANGUAGE = (index: number) => Object({
-    type: CONFIG_LANGUAGE_CHANGE, index: index
+export const CHANGE_LANGUAGE = (value: any) => Object({
```
Comment (dternyak) [inline common/actions/config.js]: why not `value: string`?

Comment (crptm) [inline common/actions/config.js]: Its moving whole objects to save some typing, could change later

Comment (dternyak) [inline common/actions/config.js]: gotcha

File: common/containers/App/index.jsx [hunk 2]
```diff
@@ -87,24 +72,18 @@ function mapStateToProps(state) {
...
 }
 
 function mapDispatchToProps(dispatch) {
     return {
         changeNode: (i: number) => {
-            dispatch(CHANGE_NODE(i))
-        },
-        toggleNodeDropdown: () => {
-            dispatch(TOGGLE_NODE_DROPDOWN())
+            dispatch(CHANGE_NODE(i));
```
Comment (dternyak) [inline common/containers/App/index.jsx]: Is this right? Thought we were passing the whole object now? should be changed to `i: any`?

Comment (crptm) [inline common/containers/App/index.jsx]: Indeed, file has no flow declation so wasnt catched :)

**Comment by dternyak** [comment]
Really good refactors :) thanks for this.

**Pull request merged**